sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -811,6 +811,25 @@ void CAFMaker::produce(art::Event& evt) noexcept {
       }
     }
 
+    // Get the stubs!
+    art::FindManyP<sbn::Stub> fmSlcStubs =
```

Comment:
If the Stub object is always in a one-to-one relationship with slices, ought we to be using `FindOneP` here?

Reply:
Stubs are not in a one-to-one relationship with slices.

Reply:
I'm looking at the `.at(0)` below. I guess I mean that each slice has either zero or one stubs, rather than literally one-to-one.

Reply:
It's because the slice list passed to the FindManyP only has one element.

Reply:
:facepalm: got it